frontend/packages/dev-console/src/components/pipelines/modals/common/utils.ts

```diff
@@ -76,6 +76,7 @@ export const getPipelineRunData = (
       resources,
       ...(params && { params }),
       workspaces,
+      status: null,
```

Comment:
Looks like the [only purpose](https://github.com/tektoncd/pipeline/blob/e153c6f2436130e95f6e814b4a792fb2599c57ef/pkg/apis/pipeline/v1beta1/pipelinerun_types.go#L149-L151) of this field is currently to cancel the Pipeline. I think this is a fine solution.